_posts/2018-04-17-making-a-components-pod.markdown

```diff
@@ -38,26 +38,32 @@ minutes, running though it properly should take about an hour.
 
 ## What Are We Going To Do?
 
-1.  We will use `pod lib create` to make a library repo
+We will make a React Native components library (GitDawg) for an existing open-source iOS Swift Application
+(GitHawk). This reflects an existing app with a team who wants to experiment with React Native in a way that doesn't
+affect the main codebase. In Artsy, GitDawg is [Emission][emission], and GitHawk is [Eigen][eigen].
 
-2.  We will use `react-native init` to make a React Native environment
+1.  We will clone and set up GitHawk
 
-3.  We will bundle the React Native code into our Pod's asset folder
+2.  We will use `pod lib create` to make a library repo called GitDawg
 
-4.  We will edit the Podspec for GitDawg, and then the Podfile for the example project to consume it
+3.  We will use `react-native init` to make a React Native environment
 
-5.  We will use [cocoapods-fix-react-native][cpfrn] to hotfix the React Native files
+4.  We will bundle the React Native code into our Pod's asset folder
 
-6.  We will expose a UIViewController which corresponds to the default screen from `react-native init` in our Pod
+5.  We will edit the Podspec for GitDawg, and then the Podfile for the example project to consume it
 
-7.  We will change the storyboard reference to point to the UIViewController from our Pod, and run the simulator to
+6.  We will use [cocoapods-fix-react-native][cpfrn] to hotfix the React Native files
+
+7.  We will expose a UIViewController which corresponds to the default screen from `react-native init` in our Pod
+
+8.  We will change the storyboard reference to point to the UIViewController from our Pod, and run the simulator to
     see our welcome screen.
 
-8.  We will set up GitHawk to consume our new Pod
+9.  We will set up GitHawk to consume our new Pod
 
-9.  We will change GitHawk to show our new UIViewController instead of the bookmarks screen
+10. We will change GitHawk to show our new UIViewController instead of the bookmarks screen
 
-10. We will edit GitDawg to act more like a development environment
+11. We will edit GitDawg to act more like a development environment
 
 ## GitHawk
```

Comment:
Okay, GH is not letting me comment on line 77. But basically, after GitHawk, you start on creating GitDawg. Where are we at this point? Still within the GitHawk folder? Or do we navigate elsewhere / create a dedicated folder?

Reply:
You should go back a folder, I'll make a note